domains/mrksbgg.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "mrksbgg",
+           "email": "lapo.ez.yt@gmail.com",
+           "discord": "1097383143708827679"
+        },
+    
+        "record": {
+            "A": ["83.69.106.193"]
```

Comment:
I cannot reach your website, keeps on loading

Reply:
Should be up now, check again please